Makefile

```diff
@@ -11,20 +11,24 @@ UTIL=Util/*.cpp
 
 EXECUTABLE=smashbot
 
-all: goals strats tactics chains main util
+all: util goals strats tactics chains main
```

Comment:
Putting it at the start doesn't guarantee it runs before the others. If you do `-j4` or whatever it might run them concurrently and then you won't have `constants.h`.